src/lean_spec/subspecs/containers/signature.py

```diff
@@ -0,0 +1,21 @@
+"""Signature container."""
+
+from __future__ import annotations
+
+from lean_spec.types import Bytes3100
+
+
+class Signature(Bytes3100):
+    """Represents aggregated signature produced by the leanVM (SNARKs in the future)."""
+
+    @staticmethod
+    def _is_valid_signature(signature: Bytes3100) -> bool:
```

Comment:
```suggestion
    def _is_valid_signature(signature: Signature) -> bool:
```
I think we want the parameter type be Signature?